lib/index.ts

```diff
@@ -289,9 +289,8 @@ export default class Backburner {
   /**
    * Schedule the passed function to run inside the specified queue.
    */
-  public schedule(queueName: string, method: Function);
+  public schedule(queueName: string, method: () => any);
   public schedule<T, U extends keyof T>(queueName: string, target: T, method: U, ...args);
-  public schedule(queueName: string, target: any | null, method: any | Function, ...args);
```

Comment:
removed because I was getting 
`ERROR: backburner.js/lib/index.ts[293, 78]: variable name clashes with keyword/type`

Reply:
Which variable name is it complaining about?

Reply:
method parameters I think